docs/Troubleshooting.md

```diff
@@ -57,5 +65,25 @@ Error: Error creating NetApp Account "sse-vdsdp-ha1-netappaccount" (Resource Gro
   29: resource "azurerm_netapp_account" "anf" {
  ```
 
- **Resolution:**
+ ### Resolution:
  Check your Azure Subscription has been granted access to Azure NetApp Files service: [Azure Netapp Quickstart](https://docs.microsoft.com/en-us/azure/azure-netapp-files/azure-netapp-files-quickstart-set-up-account-create-volumes?tabs=azure-portal#before-you-begin)
+
+
+## Azure NetApp NFSv3 volume file lock issue
+In event of SAS Viya Platform deployment shutdown on an AKS cluster with Azure NetApp NFSv3 volume, the file locks persist and `sas-consul-server` cannot access raft.db until the file locks are broken. 
+
+### Resolution:
+There are two options to avoid this issue:
+
+1. Break the file locks from Azure Portal. For details see [Troubleshoot file locks on an Azure NetApp Files volume](https://learn.microsoft.com/en-us/azure/azure-netapp-files/troubleshoot-file-locks).
+
+2. Use Azure NetApp NFS volume version 4.1. This can be done by adding the variable `netapp_protocols` to your terraform.tfvars.
+
+   **Note:** Changing this on existing cluster will result in data loss.
+   
+   Example:
+   ```bash
+   # Storage HA
+   storage_type = "ha"
+   netapp_protocols = ["NFSv4.1"]
```

Comment:
The netap protocol will always be nfs version 4.1 no need to have this as an option or listed here. If we're going to surface this be aware of the support implications. This should be a variable but not listed in the CONFIG-VARS.md file until we're ready to surface the functionality in full for customers and are ready to answer questions.

Reply:
I'll fix this.

Reply:
NetApp NFS protocol version default is set to v4.1, updated the wording to reflect the same.